### PR TITLE
Change 1.10 PRM to @MaciekPytel

### PR DESCRIPTION
**Pull request by shyamjvs**
As unfortunately, I wouldn't be able to make it for this release.

/cc @MaciekPytel @wojtek-t

### CODE REVIEW

**Comment by shyamjvs** [comment]
cc @kubernetes/kubernetes-release-managers

File: releases/release-1.10/release_team.md [hunk 1]
```diff
@@ -9,5 +9,5 @@
 | Docs | Jennifer Rondeau ([@Bradamant3](https://github.com/Bradamant3)) | Jessica Yao ([@abiogenesis-now](https://github.com/abiogenesis-now)) |
 | Release Notes | Nick Chase ([@nickchase](https://github.com/nickchase)) | Noah Abrahams ([@qedrakmar](https://github.com/qedrakmar)) |
 | Communications | Natasha Woods (@nwoods)  | Caitlyn O'Connell |
-| Patch Release Manager | Shyam JVS ([@shyamjvs](https://github.com/shyamjvs)) | |
+| Patch Release Manager | Maciek Pytel ([@MaciekPytel](https://github.com/shyamjvs)) | |
```
Comment (MaciekPytel) [inline releases/release-1.10/release_team.md]: Can you change the link?

**Comment by MaciekPytel** [comment]
Just in case - we discussed this with @shyamjvs and I volunteered for the role.

File: releases/release-1.10/release_team.md (hunk 1 above)
Comment (shyamjvs) [reply releases/release-1.10/release_team.md]: Oh that's a shame.. fixed it.

**Comment by idvoretskyi** [comment]
cc @jdumars

**Comment by jdumars** [comment]
Thank you for the heads up, and thank you @MaciekPytel for your service!
/approve

**Comment by shyamjvs** [comment]
@jdumars @idvoretskyi Could one of you merge this PR if you have write access? Thanks.

**Pull request merged**